crates/bevy_render/src/color.rs

```diff
@@ -26,13 +26,44 @@ pub struct Color {
 unsafe impl Byteable for Color {}
 
 impl Color {
+    pub const ALICEBLUE: Color = Color::rgb_linear(0.94, 0.97, 1.0);
+    pub const ANTIQUEWHITE: Color = Color::rgb_linear(0.98, 0.92, 0.84);
```

Comment:
can we "rustify" these multi-word constants? Ex: `ANTIQUEWHITE` -> `ANTIQUE_WHITE`

Reply:
Sure